front-end/lambda-in/src/Components/Landing Page/Image Container/image-container.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import './image-container.css';
+import Button from '@material-ui/core/Button';
+
+class ImageContainer extends React.Component {
```

Comment:
I would suggest to name the file in CamelCase, so the file will match the Class name.
```
ImageContainer.js
```